We cannot generate bindings for Nfc or RemoteObjects because they
haven't been packaged for Gentoo yet. On the other hand, the modules:
Script, ScriptTools and XmlPatterns have long since been deprecated.

--- a/src/github.com/therecipe/qt/internal/binding/parser/helper.go
+++ b/src/github.com/therecipe/qt/internal/binding/parser/helper.go
@@ -191,10 +191,8 @@
 	"AndroidExtras": {"Core"},
 	"Gui":           {"Widgets", "Core"}, //Widgets
 	"Network":       {"Core"},
-	"Xml":           {"XmlPatterns", "Core"}, //XmlPatterns
+	"Xml":           {"Core"}, //XmlPatterns
 	"DBus":          {"Core"},
-	"Nfc":           {"Core"},
-	"Script":        {"Core"},
 	"Sensors":       {"Core"},
 	"Positioning":   {"Core"},
 	"Widgets":       {"Gui", "Core"},
@@ -202,7 +200,6 @@
 	"MacExtras":     {"Gui", "Core"},
 	"Qml":           {"Network", "Core"},
 	"WebSockets":    {"Network", "Core"},
-	"XmlPatterns":   {"Network", "Core"},
 	"Bluetooth":     {"Core"},
 	"WebChannel":    {"Network", "Qml", "Core"}, //Network (needed for static linking ios)
 	"Svg":           {"Widgets", "Gui", "Core"},
@@ -210,7 +207,6 @@
 	"Quick":         {"QuickWidgets", "Widgets", "Network", "Qml", "Gui", "Core"}, //QuickWidgets, Widgets, Network (needed for static linking ios)
 	"Help":          {"Sql", "Network", "Widgets", "Gui", "Core"},                 //Sql + CLucene + Network (needed for static linking ios)
 	"Location":      {"Positioning", "Quick", "Gui", "Core"},
-	"ScriptTools":   {"Script", "Widgets", "Core"}, //Script, Widgets
 	"UiTools":       {"Widgets", "Gui", "Core"},
 	"X11Extras":     {"Gui", "Core"},
 	"WinExtras":     {"Widgets", "Gui", "Core"},
@@ -237,7 +233,6 @@
 	"WebView":  {"Core"},
 
 	"NetworkAuth":   {"Network", "Gui", "Core"},
-	"RemoteObjects": {"Network", "Core"},
 
 	"WebKit": {"WebKitWidgets", "Multimedia", "Positioning", "Widgets", "Sql", "Network", "Gui", "Sensors", "Core"},
 
@@ -389,8 +384,6 @@
 		"Network",
 		"Xml",
 		"DBus",
-		"Nfc",
-		"Script", //depreached (planned) in 5.6
 		"Sensors",
 		"Positioning",
 		"Widgets",
@@ -398,7 +391,6 @@
 		"MacExtras",
 		"Qml",
 		"WebSockets",
-		"XmlPatterns",
 		"Bluetooth",
 		"WebChannel",
 		"Svg",
@@ -406,7 +398,6 @@
 		"Quick",
 		"Help",
 		"Location",
-		"ScriptTools", //depreached (planned) in 5.6
 		"UiTools",
 		//"X11Extras", //TODO:
 		"WinExtras",
@@ -433,7 +424,6 @@
 		"WebView",
 
 		//"NetworkAuth", //TODO:
-		"RemoteObjects",
 
 		"WebKit",
 
